smtp.php

```diff
@@ -0,0 +1,21 @@
+<?php
+/**
+ * Plugin Name: SMTP
+ * Description: Send mail through SMTP.
+ *
+ * Plugin URI: https://github.com/trendwerk/smtp
+ *
+ * Author: Trendwerk
+ * Author URI: https://github.com/trendwerk
+ *
+ * Version: 1.1.0
```

Comment:
I meant the version number should not be specified at all in a PR. Changes the code does not necessarily involve releasing it. However, if you insist on changing it for this PR, you should make it `1.0.2` according to semver.